### DISCUSSION TITLE
Include Open WebUI in an existing React application

### DISCUSSION THREAD

**Discussion by martin-prillard** [open-webui/open-webui#7010] [General]
Hello,

Is it possible to integrate / embed Open WebUI in an existing application, like a React component?
For example, to add a chat module to an existing website with the UI of Open WebUI.

As is the case with Chainlit : https://docs-prerelease.chainlit.io/customisation/react-frontend

**Comment by anthonylmason** [comment]
@tjbck any plans around looking into this? I'm not seeing anything about it on the roadmap, and was curious since we're looking to do the same thing for our Angular application.

**Comment by revskill10** [comment]
Any news ? Without this, it seems useless for integration.

**Comment by mahimairaja** [comment]
Yes if we could easily integrate it would be nice
